packages/ui/src/theme/tokens/components/button.ts

```diff
@@ -133,8 +160,160 @@ export const button: Required<ButtonTokens<'default'>> = {
     backgroundColor: { value: 'transparent' },
     borderColor: { value: '{colors.border.tertiary.value}' },
   },
-
   // variations
+  outlined: {
+    info: {
+      borderColor: { value: '{colors.blue.60.value}' },
+      backgroundColor: { value: 'transparent' },
+      color: { value: '{colors.blue.100}' },
+      _hover: {
+        borderColor: { value: '{colors.blue.60.value}' },
+        backgroundColor: { value: '{colors.blue.10.value}' },
+        color: { value: '{colors.blue.100.value}' },
+      },
+      _focus: {
+        borderColor: { value: '{colors.blue.100.value}' },
+        backgroundColor: { value: '{colors.blue.10.value}' },
+        color: { value: '{colors.blue.100.value}' },
+        // Should we make field control box shadows for info, warning, success?
+        boxShadow: {
```

Comment:
Oh good catch. Fixed in the latest commit [here](https://github.com/aws-amplify/amplify-ui/pull/4266/commits/cdd6ca7de4363efefa825c4c27e6f960d4b1b950) Also deployed to preview docs site but it might still be building depending on when you see this :)